lib/rubygems/commands/cleanup_command.rb

```diff
@@ -10,11 +10,17 @@ def initialize
           :force => false, :install_dir => Gem.dir,
           :check_dev => true
 
-    add_option('-n', '-d', '--dryrun',
+    add_option('-n', '-d', '--dry-run',
                'Do not uninstall gems') do |value, options|
       options[:dryrun] = true
     end
 
+    add_option(:deprecated, '--dryrun',
```

Comment:
Could you fix the casing to `:Deprecated`? Output looks better that way ("Deprecated Options" instead of "deprecated Options").